release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1413,6 +1413,30 @@ sourceStrategy:
+
+* Previously, the `corespersocket` value could be higher than the `numCores` value when installing a VMware cluster using installer-provisioned infrastructure. This could cause unexpected results during installation. This update corrects this behavior by warning the user to correct these values before the cluster is created. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2034147[*BZ#2034147*])
+
+* Previously, a rare bug caused inconsistency during AWS cluster installations. The installation program has been updated to avoid this scenario. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2046277[*BZ#2046277*])
+
+* Previously, the bootstrap machine used a default size and instance type when installing on Azure using installer-provisioned infrastructure. With this update, the bootstrap machine uses the size and instance type of the control plane machines. You can now control the size and instance type of the bootstrap machine by modifying the control plane settings in your installation configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2026356[*BZ#2026356*])
+
+* Previously, the installation program provided Terraform with an ambiguous network name. This could lead to Terraform being unable to determine the correct network to use. With this update, the installation program provides Terraform with a unique network ID so that the installation succeeds. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918005[*BZ#1918005*])
+
+* Previously, control plane machines could be created before the dependent NAT gateway was created when installing a cluster on AWS, causing the installation to fail. With this update, terraform ensures that the NAT gateway has been created before the control plane machines are created. This ensures that the installation will succeed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049108[*BZ#2049108*])
+
+* Previously, if you used an OVN network rather than the default OSN network, the scale-up task failed because it took longer than the maximum amount of time required. This update doubles the amount of retries during the scale-up task so that the task can be completed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2090151[*BZ#2090151*])
+
+* Previously, when you tried to delete multiple clusters from the database in parallel, the delete process failed because of a bug in the `vmware` and `govmomi` libraries. The bug caused one of the cluster's tags to be deleted, which resulted in a 404 error when the delete process could not find the tag. This update ignores tags that aren't found and continues the delete process so that it finishes without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021041[*BZ#2021041*])
```

Comment:
```suggestion
* Previously, when you tried to delete multiple clusters from the database in parallel, the deletion process failed because of a bug in the `vmware` and `govmomi` libraries. The bug caused one of the cluster's tags to be deleted, which resulted in a 404 error when the deletion process could not find the tag. This update ignores tags that aren't found and continues the deletion so that it finishes without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021041[*BZ#2021041*])
```